packages/build/src/npm-packages/bump.ts

```diff
@@ -48,3 +56,19 @@ export async function bumpNpmPackages(version: string): Promise<void> {
     );
   }
 }
+
+/** Bumps independent packages without setting a new version of mongosh. */
+export function bumpIndependentPackages() {
```

Comment:
independent = auxiliary, right?

Reply:
Yeah this ended up going through many names and forgot to rename this one; will change to auxiliary